Cargo.toml

```diff
@@ -27,17 +32,17 @@ blst = { version = "0.3.11", default-features = false }
 hex = { version = "0.4.2", default-features = false, features = ["alloc"] }
 libc = { version = "0.2", default-features = false }
 serde = { version = "1.0", optional = true, default-features = false, features = [
-    "alloc",
-    "derive",
+	"alloc",
+	"derive",
 ] }
 
 [dev-dependencies]
 criterion = "0.5.1"
 glob = "0.3.1"
 rand = "0.8.5"
 serde = { version = "1.0", features = ["derive"] }
-serde_yaml = "0.9.17"
 serde_json = "1.0.105"
+serde_yaml = "0.9.17"
```

Comment:
same question w/ reordering the deps here

Reply:
I think this is a good change. It was alphabetically out-of-order before.